src/connectors/h1_collect.py

```diff
@@ -202,73 +209,50 @@ def paginated_insert_reports(landing_table, options, dryrun):
     api_identifier = options['api_identifier']
     api_token = options['api_token']
     program_name = options['program_name']
+    next_exists = True
+    recorded_at = datetime.utcnow()
 
-    response = load_data(
-        'https://api.hackerone.com/v1/reports',
-        api_token,
-        api_identifier,
-        params={
-            'filter[program][]': program_name,
-            'page[size]': PAGE_SIZE,
-            'page[number]': page_number
-        },
-    )
-    reports = response.json()['data']
-    recorded_at = parsedate_to_datetime(response.headers.get('Date'))
```

Comment:
let's keep this, and move it to the `load_data` function so that it doesn't accidentally get dropped again